src/govuk/components/character-count/character-count.mjs

```diff
@@ -367,14 +374,11 @@ export default CharacterCount
 
 /**
  * @typedef {object} CharacterCountTranslations
- * @property {PluralisedTranslation} [charactersUnderLimit] - Characters under limit
+ * @property {import('../../i18n.mjs').PluralisedTranslation} [charactersUnderLimit] - Characters under limit
```

Comment:
Are these changes related?

Reply:
Good spot, that was a poor conflict resolution from rebasing `main`.